Some distros are out of support for normal customers but continue
to be supported through LTSS contracts for others. Hence the EOL
date in the osinfo-db file may be past but we want the distro to
continue to show up in the list.

Index: virt-manager-2.2.0/virtinst/osdict.py
===================================================================
--- virt-manager-2.2.0.orig/virtinst/osdict.py
+++ virt-manager-2.2.0/virtinst/osdict.py
@@ -445,7 +445,7 @@ class _OsVariant(object):
 
         # If no EOL is present, assume EOL if release was > 5 years ago
         if rel is not None:
-            rel5 = _glib_to_datetime(rel) + datetime.timedelta(days=365 * 5)
+            rel5 = _glib_to_datetime(rel) + datetime.timedelta(days=365 * 10)
             return now > rel5
         return False
 
